Warhammer 4e Character Sheet/Warhammer 4e Character Sheet.html

```diff
@@ -10363,11 +10374,1842 @@ <h4 class="sheet-center">
+	}; break;
+case "spring": spell = {
+	"Name": "Spring",
+	"Range": "Touch",
+	"Target": "Special",
+	"Duration": "Willpower Bonus Rounds",
+	"Desc": "You touch the ground and water bubbles forth at the rate of 1 pint per Round, to a total of your Initiative Bonus in pints."
+	}; break;
+case "shock": spell = {
+	"Name": "Shock",
+	"Range": "Touch",
+	"Target": "1",
+	"Duration": "Instant",
+	"Desc": "Your target receives 1 Stunned Condition."
+	}; break;
+case "sly_hands": spell = {
+	"Name": "Sly Hands",
+	"Range": "You",
+	"Target": "You",
+	"Duration": "Willpower Bonus Rounds",
+	"Desc": "You teleport a small object — no bigger than your fist — from about your person into your hand."
+	}; break;
+case "sounds": spell = {
+	"Name": "Sounds",
+	"Range": "Willpower yards",
+	"Target": "Special",
+	"Duration": "Willpower Bonus Rounds",
+	"Desc": "You create small noises nearby. You can create quiet, indistinct noises that sound as if they come from a specific location within range, regardless of line of sight. The noises can evoke something specific, such as footsteps, whispers or the howl of an animal, but nothing so distinct that it might convey a message.\nWhile the spell is active, you may control the sounds by passing a Average (+20) Channelling Test. A success allows you to move the sounds to another point within range, or to increase or decrease their volume."
+	}; break;
+case "twitch": spell = {
+	"Name": "Twitch",
+	"Range": "Willpower Bonus yards",
+	"Target": "Special",
+	"Duration": "Instant",
+	"Desc": "You cause a small object to move, slightly. Something may fall from a shelf, or a book may slam its pages shut. If the object is held, the holder must pass an Average (+20) Dexterity Test or drop the object."
+	}; break;
+case "warning": spell = {
+	"Name": "Warning",
+	"Range": "1 yard",
+	"Target": "Special",
+	"Duration": "Instant",
+	"Desc": "You channel magic into an object, noticing immediately if it has been poisoned or trapped."
+	}; break;
+}
+setAttrs({
+repeating_spellbookpetty_spellname: spell.Name,
+repeating_spellbookpetty_spellrange: spell.Range,
+repeating_spellbookpetty_spelltarget: spell.Target,
+repeating_spellbookpetty_spellduration: spell.Duration,
+repeating_spellbookpetty_spelldescription: spell.Desc
+});
+});
 
+on(`change:repeating_spellbookarcane:spelldrop`, function(eventInfo) {
+var dropValue = eventInfo.newValue;
+var spell = { Name: "", "Lore": "", "CN": "0", "Range": "", "Target": "", "Duration": "", "Desc": "" };
+switch(dropValue) {
+case "aethyric_armour": spell = {
```

Comment:
Is this content available through an OGL source? Character sheets cannot have copy written content.

- Cassie

Reply:
Your right, no OGL at this time. text removed from code.